source/_integrations/wallbox.markdown

```diff
@@ -25,26 +25,28 @@ The **Wallbox** {% term integration %} pulls data from the [MyWallbox Portal](ht
 
 The {% term integration %} adds the following sensors:
 
-- Added energy
-- Added range
-- Charging power
+- Added energy (kWh)
+- Added range (km)
+- Charging power (kW)
 - Charging speed
-- Charging time
-- Cost
-- Current mode
-- Depot price
-- Energy price
-- Max available power
-- State of charge
-- Status description
-- Max charging current
+- Charging time 
+- Cost ([default currency])
+- Current mode 
+- Depot price ([default currency]/kWh)
+- Energy price ([default currency]/kWh)
+- Max available power (A)
```

Comment:
```suggestion
- Max available power (kW)
```
would this be kW?

Reply:
no, doublechecked, it's in A (25A for me, which is reasonable in A and not in kWh). The name is doubtful I agree, but it is the name that I believe Wallbox provides